neqo-transport/src/connection/mod.rs

```diff
@@ -742,6 +760,10 @@ impl Connection {
         let lost = self.loss_recovery.timeout(now);
         self.handle_lost_packets(&lost);
         qlog::packets_lost(&mut self.qlog, &lost);
+
+        if self.release_resumption_token_timer.is_some() {
+            self.create_resumption_token(now);
+        }
```

Comment:
Do we need to add this to `next_delay()`?  If we don't then it's worth explicitly noting that and the reason why.  We should ensure that all of our timers are accounted for in both places.

Reply:
I did not add it to next_delay because I did not want to trigger a process() call only for a token, but this will not be trigger that often so it is ok to add it to next_delay().